cobra/internal/base/moimpl.py

```diff
@@ -447,6 +483,31 @@ def __init__(self, parentMoOrDn, markDirty, *namingVals,
             # pylint:disable=protected-access
             self.__parentMo.__modifyChild(self, attach=True)
 
+    def clone(self, parentMo=None):
```

Comment:
I know this is merging an upstream change, but for the sake of discussion...

Do we know what the use case for this looks like? We can go with this change, but maybe we should have a todo to look at instead implementing __copy__ and __deepcopy__ as per the copy interface, and then let a clone method leverage that implementation